src/copy-filter.js

```diff
@@ -8,6 +8,8 @@ const prune = require('./prune')
 const targets = require('./targets')
 
 const DEFAULT_IGNORES = [
+  '/package-lock\\.json/',
+  '/yarn\\.lock/',
```

Comment:
In the very unlikely event that this is part of a larger filename...
```suggestion
  '/package-lock\\.json$/',
  '/yarn\\.lock$/',
```

Reply:
I initially went for `^/package-lock\\.json$`, would that help here or is end of match more important than start>

Reply:
Added